crontab/ImportPGCatalog.inc

```diff
@@ -87,16 +89,21 @@ class ImportPGCatalog extends BackgroundJob
         }
 
         echo "Extracting files from $local_compressed_file to {$this->local_catalog_dir}...\n";
-        $cmd = join(" ", [
+        $process = new Process([
             "tar",
             "--extract",
             "--bzip2",
-            "--file=" . escapeshellarg($local_compressed_file),
-            "--strip-components=3",
-            "--directory=" . escapeshellarg($this->local_catalog_dir),
+            "--file",
+            $local_compressed_file,
+            "--strip-components",
+            3,
+            "--directory",
+            $this->local_catalog_dir,
             "--overwrite",
         ]);
-        system($cmd, $ret);
+        $process->setTimeout(600);
```

Comment:
May make sense to store in some variable and name it so it is clear why and how long it is, like:
```
const MAXIMUM_EXTRACT_TIMEOUT_IN_MS = 600
```
any reason we decided on 600? Was that the default for system before perhaps?

Reply:
We already had a 300s timeout for the job and I centralized the value so its used in both places via a private class value, good suggestion.